### DISCUSSION TITLE
If Devices are Part of a Virtual Chassis, Only Show Master?

### DISCUSSION THREAD

**Discussion by adparis99** [mattieserver/netbox-topology-views#355] [General]
Hello! I presented this feature request a while back in #218. Just wanted to float it again as I still think it would be a beneficial feature for many. If a set of devices are part of a virtual chassis, the idea is that:

- Only one member device (the VC master) of the chassis would be shown on the topology
- The device would display the name of the chassis, not itself
- The device would show all interfaces/connections belonging to the chassis, not just itself

Hopefully that makes more sense. Some other thoughts:

- I originally imagined it could be possible because of the device.vc_interfaces() method that can grab all interfaces of a virtual chassis that a device is associated with
- I'm not sure if that would also allow for getting connections of said interfaces
- I'm not sure if this goes against the idea of the plugin; I know it's been a mostly physical representation to this point

**Comment by dreng** [comment]
Hi,

I wonder if you've seen FR #33 which has been accepted. It's just that someone has to take work on it. Any volunteers?

**Reply by adparis99** [reply]
Whoops, I definitely missed that FR. That said, I'm glad it's been accepted.
